azerothcore/azerothcore-wotlk
# fix(DB/Creature): Corrected Movement Speed and Spawntimer for Terrowu…

…lf Packleader


## Changes Proposed:
-  Change Movement Speed for Terrowulf Packleader to "Blizzlike" 1 (instead of 1,56)
-  Change Respawntimer for Terrowulf Packleader to "Blizzlike" 21h (Instead of a couple of mins)

## Issues Addressed:
- Closes : https://github.com/azerothcore/azerothcore-wotlk/issues/7850

## SOURCE:
- Research on WoWHead and Ingame on another Core

## Tests Performed:
- Did NOT tested it yet, but there is no reason it is not working. :)

## How to Test the Changes:

1. Update the Database to the newest Version with this fix.
2. Hop Ingame and teleport to the NPC in Ashenvale
3. Wait 21 hours for a Respawn (lol, dont do it) and check the new Walking speed ^^

## Known Issues and TODO List:
- None (Not tested!)

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
